go.mod

```diff
@@ -8,19 +8,21 @@ require (
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/gogo/protobuf v1.3.1
+	github.com/gojuno/minimock v1.9.2 // indirect
 	github.com/gojuno/minimock/v3 v3.0.6
 	github.com/google/gofuzz v1.0.0
 	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
 	github.com/insolar/assured-ledger/ledger-core/v2 v2.0.0-20200512113104-4973d6ba44e9
 	github.com/insolar/insconfig v0.0.0-20200430133349-77f6f1624abf
-	github.com/insolar/insolar v1.5.2
+	github.com/insolar/insolar v1.6.3-0.20200708160037-17e197fec1e7
 	github.com/insolar/spec-insolar-block-explorer-api v1.2.0
 	github.com/jinzhu/gorm v1.9.12
 	github.com/kelindar/binary v1.0.9
 	github.com/kr/pretty v0.2.0 // indirect
 	github.com/labstack/echo/v4 v4.1.16
 	github.com/mitchellh/mapstructure v1.3.2 // indirect
 	github.com/ory/dockertest/v3 v3.5.2
+	github.com/ory/go-acc v0.2.3 // indirect
```

Comment:
there were issues with version compatibility when installing go-acc via makefile (thtough versions are the same), so this line should be removed

Reply:
I do not know. It is all up to go mod vendor.